mldsa/poly.c

```diff
@@ -320,6 +327,16 @@ uint32_t mld_poly_chknorm(const mld_poly *a, int32_t B)
     invariant((t == 0) == array_abs_bound(a->coeffs, 0, i, B))
   )
   {
+    /*
+     * Since we know that a is in [-REDUCE32_RANGE_MAX+1, REDUCE32_RANGE_MAX-1],
```

Comment:
What does the [A, B] notation mean? Inclusive bounds? If so, I don't think you need the +1 on the lower bound?

Reply:
Yes, you are right. I eliminated the [ ] notation and used explicit <= and < to make this both correct and easier to follow.